### DISCUSSION TITLE
Can we add a style Class from a setter?

### DISCUSSION THREAD

**Discussion by dbriard** [AvaloniaUI/Avalonia#13333] [Q&A] [answered]
I have a control containing a Text box, my textbox have a class called underline to give another look to the textbox.

I would like to have the underline class to the container control, and that this class set/add the underline class to the textbox.

If it is possible, what is the syntax of the setter?

Thanks

**Comment by rabbitism** [comment, marked as answer]
https://github.com/AvaloniaUI/Avalonia/discussions/9060

It is discussed here.

**Reply by dbriard** [reply]
Thanks, so I got my answer. I'll try to find the least ugly workaround.
